tests/pubsub/utils.nim

```diff
@@ -26,9 +26,9 @@ type
 
 proc getPubSubPeer*(p: TestGossipSub, peerId: PeerId): PubSubPeer =
   proc getConn(): Future[Connection] =
-    p.switch.dial(peerId, GossipSubCodec)
+    p.switch.dial(peerId, GossipSubCodec_12)
```

Comment:
What happens if dialing 1.2.0 fails? Should we attempt to dial with a lower version?

Reply:
This is only for tests. Production code tries all versions.